package.json

```diff
@@ -23,7 +23,6 @@
         "@vscode/codicons": "0.0.20",
         "@vscode/webview-ui-toolkit": "^1.4.0",
         "antd": "^5.22.1",
-        "primeflex": "^3.3.1",
```

Comment:
`yarn.lock` needs update, too. But we can push this with #3 to get this PR here over the line.